recipes/pantax/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pantax" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/LuoGroup2023/PanTax/releases/download/v{{ version }}/{{ name }}.tar.gz  
+    sha256: c14a46b41f580cf39ad502049d267cd31bbfd4dc05028215cc669c0e52f58ce7
+
+  - url: https://files.pythonhosted.org/packages/07/0f/6039cf6e22f9cbec57cdedff103949e4856c18fd2cb714bc70cdc8b52941/gurobipy-11.0.2-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl 
```

Comment:
Please require this as a dependency and do not pip install them

Reply:
Gurobipy can be install with `conda install gurobi::gurobi` and pip. But I don't know how to set channels `gurobi` to install it.

Reply:
You can use only dependencies from `conda-forge` and `bioconda` channels. You cannot install from third-party channels and/or Pip